src/lib/signature.ts

```diff
@@ -39,6 +46,20 @@ class PrivateKey extends CircuitValue {
     return new PrivateKey(Scalar.fromBits(bs));
   }
 
+  /**
+   * Convert this {@link PrivateKey} to a bigint
+   */
+  toBigInt() {
+    return this.s.toBigInt();
+  }
+
+  /**
+   * Create a {@link PrivateKey} from a bigint
+   */
+  static fromBigInt(sk: PrivateKeyBigint) {
```

Comment:
I think we should add to the comment that creating a private key from a `bigint` may not be secure enough if the bigint wasn't sampled correctly